ballerina/tests/55_declarative_authentication_service.bal

```diff
@@ -0,0 +1,156 @@
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+listener Listener ep55WithScopes = new (9155);
+listener Listener ep55EmptyScope = new (9255);
+
+JwtValidatorConfig jwtAuthConfig55 = {
+    issuer: "wso2",
+    audience: "ballerina",
+    signatureConfig: {
+        trustStoreConfig: {
+            trustStore: {
+                path: TRUSTSTORE_PATH,
+                password: "ballerina"
+            },
+            certAlias: "ballerina"
+        }
+    },
+    scopeKey: "scope"
+};
+
+OAuth2IntrospectionConfig oauth2IntroConfig55 = {
+    url: "https://localhost:" + oauth2AuthorizationServerPort.toString() + "/oauth2/token/introspect",
+    tokenTypeHint: "access_token",
+    scopeKey: "scp",
+    clientConfig: {
+        secureSocket: {
+            cert: {
+                path: TRUSTSTORE_PATH,
+                password: "ballerina"
+            }
+        }
+    }
+};
+
+LdapUserStoreConfig ldapUserStoreConfig = {
+    domainName: "avix.lk",
+    connectionUrl: "ldap://localhost:389",
+    connectionName: "cn=admin,dc=avix,dc=lk",
+    connectionPassword: "avix123",
+    userSearchBase: "ou=Users,dc=avix,dc=lk",
+    userEntryObjectClass: "inetOrgPerson",
+    userNameAttribute: "uid",
+    userNameSearchFilter: "(&(objectClass=inetOrgPerson)(uid=?))",
+    userNameListFilter: "(objectClass=inetOrgPerson)",
+    groupSearchBase: ["ou=Groups,dc=avix,dc=lk"],
+    groupEntryObjectClass: "groupOfNames",
+    groupNameAttribute: "cn",
+    groupNameSearchFilter: "(&(objectClass=groupOfNames)(cn=?))",
+    groupNameListFilter: "(objectClass=groupOfNames)",
+    membershipAttribute: "member",
+    userRolesCacheEnabled: true,
+    connectionPoolingEnabled: false,
+    connectionTimeout: 5,
+    readTimeout: 60
+};
+
+JwtValidatorConfigWithScopes jwtAuthConfig55WithScopes = {
+    jwtValidatorConfig: jwtAuthConfig55,
+    scopes: "write"
+};
+
+JwtValidatorConfigWithScopes jwtAuthConfig55EmptyScope = {
+    jwtValidatorConfig: jwtAuthConfig55
+};
+
+OAuth2IntrospectionConfigWithScopes oauth2config55WithScopes = {
+    oauth2IntrospectionConfig: oauth2IntroConfig55,
+    scopes: "write"
+};
+
+OAuth2IntrospectionConfigWithScopes oauth2config55EmptyScope = {
+    oauth2IntrospectionConfig: oauth2IntroConfig55
+};
+
+LdapUserStoreConfigWithScopes ldapUserStoreconfig55WithScopes = {
+    ldapUserStoreConfig: ldapUserStoreConfig,
+    scopes: "developer"
+};
+
+LdapUserStoreConfigWithScopes ldapUserStoreconfig55EmptyScope = {
+    ldapUserStoreConfig: ldapUserStoreConfig
+};
+
+FileUserStoreConfigWithScopes fileUserStoreConfig55WithScopes = {
+    fileUserStoreConfig: {},
+    scopes: "write"
+};
+
+FileUserStoreConfigWithScopes fileUserStoreConfig55EmptyScope  = {
+    fileUserStoreConfig: {}
+};
+
+@ServiceConfig {
+    auth: [
+        jwtAuthConfig55WithScopes,
+        oauth2config55WithScopes,
+        // ldapUserStoreconfig55WithScopes,
```

Comment:
ditto

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/343/commits/5d5a9b59b1f48a15996c9bd25c05d1ebff7df100